packages/v4/src/content/design-guidelines/extensions/log-viewer/log-viewer.md

```diff
@@ -0,0 +1,49 @@
+---
+id: Log viewer
+section: extensions
+---
+
+A log viewer is a preconfigured component that gives you the option to visualize your log content. Log viewer renders log output in real time in a clear and structured way.
+
+## Light theme log viewer
+
+<img src="./img/logviewer.png" alt="Log Viewer"/>
+
+Type of log dropdown menu: Allow users to switch between different types of logs.
+1. Search bar: Use to look up historical logs. The results will be highlighted in the log.
+2. Pause button: Play and stop your log content viewing, instead of scrolling through.
+3. Clear log: Clear the displayed log output.
+4. Cog: House settings such as wrapping lines, showing timestamps, and displaying line numbers.
+5. Export: Export log content.
+6. Download: Download the log file.
+7. Fullscreen: Expand log viewer to full screen
+
+## Dark theme log viewer
+We recommend using the light theme editor by default, but there is also a dark theme log viewer
+available. All log viewer functionality remains the same whether a light or dark theme is used.
+
+<img src="./img/logviewerdark.png" alt="Dark theme log viewer"/>
+
+## Usability
+Use a log viewer when:
+The user can manipulate 1 large log file or multiple log files at the same time
```

Comment:
You need to add bullets to these, you can add a bullet using a * symbol in markdown!